### ISSUE TITLE
img tags

### ISSUE THREAD

**Issue by Bayramito** [HMarzban/extension-hyperlink#3]
Can't we add hyperlinks to the img tags ?

**Comment by HMarzban** [comment]
Yes, the current mark can wrap an inline image when its parent schema permits marks.
With `HyperMultimediaKit`, set `Image: { inline: true }`, select the image, then call `setHyperlink({ href })`.
Use `editHyperlink({ newURL })` for a URL-only edit. Browser tests now cover editing, HTML round-trip, and unlinking without losing the image.
The prebuilt edit form requires text, so use a host URL form for image-only links.
See [Links on images](https://github.com/docs-plus/docs.plus/tree/main/extensions/extension-hyperlink#links-on-images).
I am closing the usage question. Please share the image schema if a block image or custom node view still fails.
